src/components/common.js

```diff
@@ -118,56 +121,165 @@ export const ButtonOutline = ({ disabled, children, ...props }) => {
   }, props), [children])
 }
 
-export const TabBar = ({ activeTab, tabNames, displayNames = {}, refresh = _.noop, getHref, getOnClick = _.noop, children }) => {
+/**
+ * Creates the primary tab bar for workspaces and workflows.
+ * Semantically, this is actually a menu of links rather than true tabs.
+ *
+ * @param activeTab The key of the active tab
+ * @param tabNames An array of keys for each tab
+ * @param displayNames An optional array of display names for each tab (otherwise the keys will be displayed)
+ * @param refresh If provided, a function to refresh the current tab
+ * @param getHref A function to get the href for a given tab
+ * @param getOnClick An optional click handler function, given the current tab
+ * @param label The ARIA label for the menu, which is required for accessibility
+ * @param tabProps Optionally, properties to add to each tab
+ * @param id Optionally, an ID to apply to the wrapper `nav` element so it can be easily found in the
+ *  integration tests. This defaults to 'tab-navigation', but you should change if it this ID would appear
+ *  more than once on a page
+ * @param children Children, which will be appended to teh end of the tab bar
+ * @param props Any additional properties to add to the container menu element
+ */
+export const TabBar = ({
```

Comment:
I think it might be time to break these two out into something like `src/components/tabBars.js`, I'd say you might as well do it in this PR personally